csi/manager.go

```diff
@@ -19,8 +19,8 @@ func GetCSIManager() *Manager {
 	return &Manager{}
 }
 
-func (m *Manager) Run(driverName, nodeID, endpoint, csiVersion, identityVersion, managerURL string) error {
-	logrus.Infof("CSI Driver: %v csiVersion: %v, manager URL %v", driverName, csiVersion, managerURL)
+func (m *Manager) Run(driverName, nodeID, endpoint, identityVersion, managerURL string) error {
+	logrus.Infof("CSI Driver: %v csiVersion: %v, manager URL %v", driverName, identityVersion, managerURL)
```

Comment:
I thought I had reviewed this one as well, must have been only the backports. I would like the csiVersion to just be named version, since it's actually the longhorn version which is unrelated to the supported csi version